public/assets/boot.js

```diff
@@ -22,6 +22,10 @@ function runWorker() {
 function prepTerm() {
   term.open(document.getElementById('consoleBox'));
   term.handler = function a(indata2) {
+    console.log("am i here ", indata2);
```

Comment:
if its not too inconvenient, could we get rid of debug prints in code before merging?